sqlalchemy_jsonapi/serializer.py

```diff
@@ -459,7 +459,7 @@ def _check_instance_relationships_for_delete(self, instance):
             check_permission(instance, rel_key, Permissions.EDIT)
 
             if rel.cascade.delete:
-
+                # pragma: no cover
```

Comment:
My model does not have any relationships that on delete cascade. I left this as no cover because I need to go back and create models that do behave this way and I am looking at sqlalchemy and how to do that.